tests/test_socket_server.py

```diff
@@ -97,6 +97,10 @@ async def test_full_command_sequence(self):
         await self.write(command="start", parameters={})
         self.data = await self.read()
         self.assertEqual(ResponseCode.OK, self.data["response"])
+        self.data = await self.read()
+
+        print(self.data)
```

Comment:
Are you sure you still want this print statement? It looks like something you might have thrown in to debug a problem.

(I'm not saying you should never have a print statement in test code, but I try to limit them, and to provide plenty of context in the message, so it's purpose is clear to others. One common use case is to report how long something takes if I do need a timeout for a slow-ish thing.)

Reply:
It was debugging code indeed. I removed it.